### ISSUE TITLE
[Accessibility] NavigationView should not announce items with "Heading Level 1" when using Narrator.

### ISSUE THREAD

**Issue by niels9001** [microsoft/microsoft-ui-xaml#5552]
**Describe the bug**
I'm not sure if this is a bug or expected behavior. It was flagged by the MS accessibility team as a bug however: [#12019](https://github.com/microsoft/PowerToys/issues/12019)

If this is to expected behavior feel free to close this issue. @dedavis6797 FYI

**Steps to reproduce the bug**
1. Add a NavigationView with a couple of items
2. Turn on Narrator

**Expected behavior**
Narrator should announce the title of the NavigationView, and not "Item name + Heading Level 1" according to [#12019](https://github.com/microsoft/PowerToys/issues/12019)

**Comment by StephenLPeters** [comment]
@chigy do you know if this is by design?

**Comment by niels9001** [comment]
Any updates on this? @chigy @StephenLPeters

**Comment by StephenLPeters** [comment]
@chigy and @karenbtlai can you weigh in?

**Comment by chingucoding** [comment]
@niels9001 Just checked this and there is a small but very important difference between the behavior you reported and the actual behavior. The entries in the navigation view are being announced with "level 1/2/3" and not "heading level 1/2/3". The first one is correct since the items are nested elements, the second behavior would be a bug but the items are not being announced as heading level but as "normal" level. The TreeView exhibits the same behavior and I would argue both cases are correct here.
